### ISSUE TITLE
Stuck on 'Grant Accessibility Access'

### ISSUE THREAD

**Issue by doowmot** [noah-nuebling/mac-mouse-fix#106]
https://cln.sh/LHCQPm

^^ As attached.

It keeps asking me to grant accessibility, even though I have. I cannot get past this page. Not sure what's happening?

The app worked great for me until about a week ago.

**Comment by noah-nuebling** [comment]
Hey Randy thanks for reaching out!

You could try:
1. Unchecking and rechecking the "Mac Mouse Fix Helper" checkbox in System Preferences
2. Following the solution for "Problem: Sometimes Mac Mouse Fix Helper won't show up in System Preferences" on [this page](https://github.com/noah-nuebling/mac-mouse-fix/discussions/101).
Even though it does show up for you, this could still fix it.

Does that work for you?

**Comment by noah-nuebling** [comment]
Oh and maybe downloading the latest version (1.0.4) off the [website](https://mousefix.org/) could help as well. 1.0.3 and below don't work properly anymore unfortunately. More on that [here](https://github.com/noah-nuebling/mac-mouse-fix/issues/95)

**Comment by doowmot** [comment]
Hi Noah,

thanks for getting back to me. I've tried all the solutions provided,
including a clean install but nothing's working.

Are other's having this problem? Or is it on my end?

**Comment by noah-nuebling** [comment]
Hey @randybeans99 I'm sorry for the late reply.

I haven't heard from anyone else that they have the same issue that you're experiencing.

I'm baffled that not even a clean install helped.

I can't really do much to track this bug down, because I can't reproduce it and it's probably got more to do with macOS than with Mac Mouse Fix, but I'll keep it in the back of my mind, and do my best to figure it out.

Until then I recommend you check out [SteerMouse](https://plentycom.jp/en/steermouse/) to replicate the button remapping capabilities of Mac Mouse Fix and [MOS](https://mos.caldis.me/) as a substitute for the smooth scrolling.

**Comment by noah-nuebling** [comment]
Well I actually saw a similar issue earlier today when playing around with a different machine, where I had several versions of Mac Mouse Fix installed at once. Unchecking and rechecking the "Mac Mouse Fix Helper" checkbox in System Preferences solved it for me. But I recon you've tried that.

**Comment by doowmot** [comment]
Hi Noah, thanks anyway.

**Comment by ryan-moeller21** [comment]
I just experienced this same issue when updating to macOS Big Sur, version 11.4. I'm on a 2018 15" Macbook Pro.

I've tried a new install, I have tried unchecking and rechecking the Mac Mouse Fix Helper checkbox, and I've tried restarting my laptop. No luck on any of these attempts so far.

**Comment by noah-nuebling** [comment]
@ryan-moeller21 Very weird indeed.

What does 'clean install' / 'new install' mean exactly? Is any of your old data preserved after that?

**Comment by noah-nuebling** [comment]
Closing this now as I assume it's been resolved. Let me know if you still have issues!
